### DISCUSSION TITLE
is it possible to add a task/routine on every new entry?

### DISCUSSION THREAD

**Discussion by netwaretcs** [statamic/cms#5750] [Q&A] [answered]
Hi, I wanted to ask if there is a way to automatically generate a user group every time I add a new entry to a specific collection.

EX:
New page slug: "about_us"
auto add user group: "about_us"

**Comment by duncanmcclean** [comment, marked as answer]
Yes, you could do it yourself with an event listener, listening for the `EntrySaved` event, then create the group from that.
